src/common/arrow/arrow_converter.cpp

```diff
@@ -76,10 +99,10 @@ void ArrowConverter::setArrowFormatForInternalID(ArrowSchemaHolder& rootHolder,
     }
     child.children = &rootHolder.nestedChildrenPtr.back()[0];
     initializeChild(*child.children[0]);
-    child.children[0]->name = copyName(rootHolder, "table");
+    child.children[0]->name = copyName(rootHolder, "offset");
     setArrowFormat(rootHolder, *child.children[0], *LogicalType::INT64());
     initializeChild(*child.children[1]);
-    child.children[1]->name = copyName(rootHolder, "offset");
+    child.children[1]->name = copyName(rootHolder, "table");
```

Comment:
Was this a bug? I saw you changed the order of `offset` and `table` here.

Reply:
It turns out it was in the wrong order initially